### DISCUSSION TITLE
Frozen Columns not working after upgrade

### DISCUSSION THREAD

**Discussion by DrastiGohel** [serenity-is/Serenity#6579] [Help]
Hello,

We had frozen column logic in our code. After migrating to latest serenity version it stopped working.
Below is code
Please let us know how to fix it

![image](https://user-images.githubusercontent.com/98172041/201933214-c0496944-15fb-4b80-ab04-de05f384705d.png)

**Comment by volkanceylan** [comment]
I don't think we had a frozen columns sample before, but the system changed in SleekGrid:

https://serenity-is.github.io/sleekgrid/examples/classic/example97-frozen-columns

Try setting the column.frozen = true in getColumns method.
